requirements.txt

```diff
@@ -12,7 +12,7 @@ matplotlib~=3.0
 numpy~=1.0
 orjson~=3.0
 packaging
-pandas>=1.0,<3.0
+pandas>=1.3.0,<3.0
```

Comment:
The `df._compute()._translate(None, None)` method signature was adopted in `pandas 1.3.0`. Should be fine to constrain the requirements a bit given that pandas 1.3.0 is over 2 years old.